packages/react/src/components/pickers/BasePicker.tsx

```diff
@@ -284,17 +298,21 @@ export class BasePicker<T extends {}, P extends IBasePickerProps<T>>
           className,
           isFocused,
           disabled,
+          hasErrorMessage: hasError,
           inputClassName: inputProps && inputProps.className,
         })
       : {
           root: css('ms-BasePicker', className ? className : ''),
+          error: css('ms-BasePicker-error', ''),
+          label: css('ms-BasePicker-label', ''),
```

Comment:
If you're not passing anything else here, you can just do:

```suggestion
          error: 'ms-BasePicker-error',
          label: 'ms-BasePicker-label',
```